src/app/body/tasks-evaluation/tasks-evaluation.component.ts

```diff
@@ -0,0 +1,174 @@
+import { Component, OnInit } from '@angular/core';
+import { AngularFireFunctions } from '@angular/fire/functions';
+import { Tasks } from 'src/app/Interface/TasksInterface';
+import { ApplicationSettingsService } from 'src/app/services/applicationSettings/application-settings.service';
+import { AuthService } from 'src/app/services/auth.service';
+import { BackendService } from 'src/app/services/backend/backend.service';
+import { NavbarHandlerService } from 'src/app/services/navbar-handler/navbar-handler.service';
+import { ToolsService } from 'src/app/services/tool/tools.service';
+
+@Component({
+  selector: 'app-tasks-evaluation',
+  templateUrl: './tasks-evaluation.component.html',
+  styleUrls: ['./tasks-evaluation.component.css']
+})
+export class TasksEvaluationComponent implements OnInit {
+
+  constructor(public navbarHandlerService: NavbarHandlerService, private functions: AngularFireFunctions, public backendService: BackendService, public applicationSettingsService: ApplicationSettingsService, public authService: AuthService, public toolsService: ToolsService) { }
+  componentName: string = "TASKS-EVALUATION";
+  tasks: Tasks[] = [];
+  showLoader: boolean;
+  selectedTeamId: string;
+  todayDate: string;
+  time: string;
+  teamIds: string[];
+  teamCurrentSprint: number;
+  filterSprintNumber: number;
+  showModalLoader: boolean;
+
+  firstInResultTaskId: string;
+  lastInResultTaskId: string;
+  prev_strt_at = [];
+  pagination_clicked_count = 0;
+
+  disable_next: boolean = false;
+  disable_prev: boolean = false;
+
+  ngOnInit(): void {
+    this.navbarHandlerService.resetNavbar();
+    this.navbarHandlerService.addToNavbar(this.componentName);
+    this.todayDate = this.toolsService.date();
+    this.time = this.toolsService.time();
+
+    this.authService.afauth.user.subscribe(data => {
+      this.authService.userAppSettingObservable.subscribe(data => {
+        if (data.AppKey) {
+          this.teamIds = this.backendService.getOrganizationTeamIds();
+          this.selectedTeamId = this.authService.getTeamId();
+          this.readTasks();   
+        }
+      });
+    });
+  }
+
+  async readTasks() {
+    this.showLoader = true;
+    this.disable_next = true;
+    this.disable_prev = true;
+    const orgDomain = this.backendService.getOrganizationDomain();
+    const callable = this.functions.httpsCallable('tasksEvaluation');
+    try {
+      const result = await callable({ mode: 'readTasksEvaluationData', OrganizationDomain: orgDomain, TeamId: this.selectedTeamId, PageToLoad: 'initial', SprintNumber: this.filterSprintNumber }).toPromise();
+      this.tasks = result.Tasks;
```

Comment:
Idea to implement pagination is awesome, let's connect over this to understand more.